manifests/service.pp

```diff
@@ -2,14 +2,14 @@
 # @api private
 class oxidized::service {
   assert_private()
-
   if $facts['service_provider'] == 'systemd' {
+    include systemd::systemctl::daemon_reload
     ::systemd::unit_file { 'oxidized.service':
       ensure  => $::oxidized::service_file_ensure,
       content => template('oxidized/oxidized.service.erb'),
       notify  => Service['oxidized'],
     }
-    Exec['systemctl-daemon-reload'] -> Service['oxidized']
+    Class['systemd::systemctl::daemon_reload'] -> Service['oxidized']
```

Comment:
This will go away once I update this module to only support Puppet >= 6.1.0. Beginning with Puppet 6.1.0 Puppet will automatically run systemctl daemon-reload.